recipes/libcap/all/conanfile.py

```diff
@@ -29,26 +30,22 @@ class LibcapConan(ConanFile):
         "fPIC": True,
         "psx_syscals": False,
     }
-    exports_sources = "patches/**"
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
 
     def validate(self):
         if self.info.settings.os != "Linux":
```

Comment:
self.settings

Reply:
@SpaceIm I must be missing something obvious here, what needs corrected?

Reply:
`self.info` is not allowed in `validate()` in conan v2 anymore.